nbproject/private/c_standard_headers_indexer.c

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
This file should not be part of this pull request -- it looks like an IDE artifact. Please delete the file & add this to the .gitignore file.

Reply:
Correct, will be removed, I think i did `git add *.c` rather than` git add crypto_config.c`". Will be removed now.